src/archive/Cargo.toml

```diff
@@ -10,10 +10,10 @@ edition = "2021"
 # local dependencies
 internet_identity_interface = { path = "../internet_identity_interface" }
 # ic dependencies
-candid = "0.8"
-ic-cdk = "0.8"
-ic-cdk-timers = "0.2"
-ic-cdk-macros = "0.6"
+candid = { version = "0.9", features = ["parser"] }
```

Comment:
What does `features = ["parser"]` do and why was it added?

Reply:
The languages team decided to split the candid library (see [here](https://github.com/dfinity/candid/blob/master/Changelog.md#breaking-changes)), or rather hide not as often used functionality behind this feature flag. In particular we require the `parser` feature to test whether the code actually implements the did file.

But we don't need the parser feature for the canister. I have now only enabled in in `dev-dependencies`. Thanks.